wikipron.py

```diff
@@ -39,7 +39,7 @@
 ]
 """
 _DIALECT_SELECTOR_TEMPLATE = (
-    'span[@class = "ib-content qualifier-content"][{dialects_text}]'
+    'span[@class = "ib-content qualifier-content" and a[{dialects_text}]]'
```

Comment:
Here's the bug for dialect selection, where we shouldn't specified the dialect text for inside the `<a>...</a>` tags.